packages/core/package.json

```diff
@@ -76,15 +84,12 @@
   "scripts": {
     "build": "yarn clean && yarn bundle",
     "build:deno": "yarn bundle:deno-prepare && yarn bundle:deno-convert && yarn bundle:deno-move && yarn bundle:deno-clean",
-    "bundle": "yarn bundle:cjs && yarn bundle:esm && yarn bundle:types && yarn bundle:bun",
+    "bundle": "yarn tsup && yarn bundle:bun && yarn",
     "bundle:bun": "node ../../config/bunnify.mjs",
-    "bundle:cjs": "tsup src --format cjs --clean -d dist/cjs --loader '.mts=ts'",
     "bundle:deno-prepare": "node ../../config/prepare-denoify.mjs",
     "bundle:deno-convert": "denoify --src _deno_prepare/src",
     "bundle:deno-move": "rimraf ../../deno_dist/core && mv deno_dist ../../deno_dist/core",
     "bundle:deno-clean": "rimraf _deno_prepare",
-    "bundle:esm": "tsc --p tsconfig.esm.json",
```

Comment:
Just to keep things clean, I think these json files can be removed too

Reply:
I'll fixed it.